redis/mutation_utils.cc

```diff
@@ -51,6 +52,29 @@ atomic_cell make_cell(const schema_ptr schema,
     return atomic_cell::make_live(type, api::new_timestamp(), value, atomic_cell::collection_member::no);
 }  
 
+// cas_request sub-class for write_strings
+class write_strings_cas_request : public service::cas_request {
+private:
+    service::storage_proxy& _proxy;
+    schema_ptr _schema;
+    bytes& _key;
```

Comment:
I'll need to look further, but I'm worried that this is a reference. Who owns this key? What happens when the request is sent to a different core? I guess I see some of the answers below as I keep reading.

Reply:
Stop using a reference here